### DISCUSSION TITLE
Unselect Tab

### DISCUSSION THREAD

**Discussion by 1Luc1** [vuetifyjs/vuetify#19716] [Q&A] [answered]
I was wondering if it is possible to un-select a tab? https://vuetifyjs.com/en/components/tabs/
 Or at least remove the selection indicator?

**Comment by m50S79sM6SRNp8Jn** [comment]
Why do you need it?

**Reply by 1Luc1** [reply]
I use tabs combined with vue-router as website navigation. The Home (/) route isn't within the tabs, so showing the Home page shouldn't select any tab.

**Comment by konstantin-karlovich-unbiased-co-uk** [comment, marked as answer]
@1Luc1 try to use empty [selected-class](https://vuetifyjs.com/en/api/v-tab/#props-selected-class)
